src/main/api/ledger.ts

```diff
@@ -19,23 +42,26 @@ export const getBTCAddress = async (network: Network) => {
     await transport.close()
     return E.right(info.bitcoinAddress)
   } catch (error) {
-    if (error.message === 'NoDevice') {
-      return E.left(LedgerErrorId.NO_DEVICE)
-    }
-    if (error.message.includes('cannot open device')) {
-      return E.left(LedgerErrorId.ALREADY_IN_USE)
-    }
-    switch (error.statusText) {
-      case 'SECURITY_STATUS_NOT_SATISFIED':
-        return E.left(LedgerErrorId.NO_APP)
-      case 'CLA_NOT_SUPPORTED':
-        return E.left(LedgerErrorId.WRONG_APP)
-      case 'INS_NOT_SUPPORTED':
-        return E.left(LedgerErrorId.WRONG_APP)
-      case 'CONDITIONS_OF_USE_NOT_SATISFIED':
-        return E.left(LedgerErrorId.DENIED)
-      default:
-        return E.left(LedgerErrorId.UNKNOWN)
+    return E.left(getErrorId(error.message, error.statusText))
+  }
+}
+
+export const getBNBAddress = async (network: Network) => {
+  try {
+    const timeout = 50000
+    const transport = await ledger.transports.node.create(timeout)
+    const app: LedgerApp = new ledger.app(transport, 100000, 100000)
+    const hdPath = [44, 714, 0, 0, 0]
```

Comment:
Just curious: Can we use same `hdPath` for `testnet` and `mainnet`?

Reply:
yes for the binance chain app in ledger